doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+Begin by initializing a SmartRedis Client object per launched database.
+
+To create a designated SmartRedis Client, you need to specify the address of the target
+running database. You can easily retrieve this address using the get_address() function
+provided by the initialized Orchestrator object.
+
+.. note::
+
+  Make sure to set the 'cluster' argument appropriately based on whether the database is in a clustered configuration or not.
+
+Here's the code example:
+
+.. code-block:: python
+  client1 = Client(address=standalone_database.get_address()[0], cluster=False)
+  client2 = Client(address=clustered_database.get_address()[0], cluster=True)
+
+In this code, 'client1' is initialized for the standalone Orchestrator, while 'client2' is
+initialized for the clustered Orchestrator.
+
+In this example, we transmit, retrieve and manipulate a NumPy tensor. Begin by creating a
+NumPy array and send it to each database using the associated SmartRedis client. To
+accomplish this, use the put_tensor() function provided by the SmartRedis Client API.
+We access this function through the SmartRedis Client objects, client1 and client2.
+In this example, we are sending the same 'array' to both databases.
+
+.. code-block:: python
+  array = np.array([1, 2, 3, 4])
+  client1.put_tensor("tensor1", array)
+  client2.put_tensor("tensor2", array)
+
+In the next stage of the workflow, we are going to write a software program
+that utilizes the previously launched databases. In SmartSim terms, the application
+is called a Model.
+
+To run a software program, you must specify the run settings: run command and
+source code file. In this example, we invoke the Python 3 interpreter to run a
+python script. Create the run settings using the create_run_settings() function
+provided by the exp (Experiment) object. Specify the path to the script to the argument
+`exe_args` and the executable to the `exe_ex` argument.
+
+.. code-block:: python
+  srun_settings = exp.create_run_settings(exe="python3", exe_args="/path/to/file.py")
+
+Next, set the number of nodes and tasks per node for the ‘Model’. For the application (Model),
+the number of nodes determines the distribution of computational tasks while tasks per node
+specifies how many individual instances of the program (tasks) can run on each node. In this
+example, we specify to SmartSim that we intend to execute the `Model` once on a single node.
+
+.. code-block:: python
+  srun_settings.set_nodes(1)
+  srun_settings.set_tasks_per_node(1)
+
+Next, create an instance of the Model object using the create_model() function of the
+exp object. Pass in the srun_settings object, which was configured earlier, as an argument
+to the create_model() function and assign to the variable ‘model’.
+
+.. code-block:: python
+  model = exp.create_model("tutorial-model", srun_settings)
+
+Next, launch the model instance using the exp.start() function.
+
+.. code-block:: python
+  exp.start(model, block=True, summary=True)
+
```

Comment:
Maybe insert a subheader here, labeling this as the beginning of the SmartRedis side?

Reply:
Could add other subheaders as well if you want to add more structural organization to the example documentation

Reply:
I am stealing that idea!